befta_recent_executions_info_UnspecifiedEnvironment.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
could you please remove this file, it is not required. please update the ignore list for git.